docs/modules/edged.md

```diff
@@ -0,0 +1,61 @@
+# EdgeD
+
+## Overview
+
+EdgeD is an edge node module which manages the POD lifecycle. With help of the POD specifications provided as JSON from Cloud via MetaManager, it ensures the containers for the POD's (with containers as described in POD spec) are launched over the edge node, are running and healthy.
+
+EdgeD interfaces with MetaManager module to receive all the configuration calls. It responds to MetaManager module with all config results and queries.
```

Comment:
It's good if we can have some block diagram to show : 
- the architecture of edged
- interfacing with what all modules
- document any interface APIs that edged exposes